eyre/tests/test_location.rs

```diff
@@ -127,6 +130,7 @@ fn test_option_compat_wrap_err() {
     println!("{:?}", err);
 }
 
+#[cfg(feature = "anyhow")]
```

Comment:
Could we change these to use `eyre::WrapErr` or `ok_or_eyre` instead to test them, and show there is a non-anyhow alternative?

Reply:
I am not sure I understand what you mean here.

To my mind, this test tests the interaction of `wrap_err_with` with `ContextCompat`.

Reply:
On a second look, this specifically tests `Option::wrap_err_with`, which is not provided by `WrapErr`.

`WrapErr` on `Result` is tested in [`fn test_wrap_err`](https://github.com/eyre-rs/eyre/pull/138/files#diff-1ff47dac6cf55e34ff587968c5b1f1ec6b6ae39d2668a66ecba3633163a21fc5R43).

`OptionExt` provides `Option::ok_or_eyre`, which is the pure-`eyre` analog to `Option::wrap_err_with` from the `anyhow` compat layer.

Reply:
Yes, I thought it may be worthwhile to show how to do the same without anyhow in another test.

Just realized that the `ContextCompat` has a `.context` method and a `.wrap_err`, which got me into thinking it just called down to `WrapErr::wrap_err` instead; which is rather inconsistent with two traits with the same method name. All the better to get this feature gated.

I can push up a test later that adds a test for `ok_or_eyre` for location, as I realized that it does not pass currently 😬 

I am not 100% familiar with all of this crate yet, so thank you for your patience

Reply:
Ah, now I see! :) 

Testing `ok_or_eyre` for location should have been part of https://github.com/eyre-rs/eyre/pull/129. I totally missed that. I, too, am not 100% familiar with all of this crate yet, and thank *you* for your patience just the same.

Reply:
https://github.com/eyre-rs/eyre/pull/140

Fixed it here, found a missing `track_caller`

Reply:
Awesome, thank you!